.github/ISSUE_TEMPLATE/qa_issue_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Manual QA
+about: Use this template for running manual QA issues.
+title: "[DATE]: [FEATURE NAME]"
+labels: qa, review
+---
+# Manual QA for # 
+
+### Check list
+- [] columns of experiments table are appropriate sizes 
+- [] order of colomns is correct  
+- []header menu has appropriate options  
+- []shadow of the first column is absent while first opening 
```

Comment:
here in the other places. let's generalize the list of issues to test a bit - "shadow - not visible initially, appears up and side when scroll, etc" I don't think we need that level of granularity. Otherwise the list could become 1000+ items eventually.